package/resources/openmrs-runtime.properties

```diff
@@ -0,0 +1,7 @@
+encryption.vector=GEurvjC9CKVoLcK8ncmkTA\=\=
+connection.url=jdbc\:mysql\://openmrsdb\:3306/openmrs?autoReconnect\=true&sessionVariables\=default_storage_engine\=InnoDB&useUnicode\=true&characterEncoding\=UTF-8&zeroDateTimeBehavior\=convertToNull
```

Comment:
Here the connection string , **credentials** are hardcoded. This needs to be refactored to use environment variables. 
Also, do we need to maintain this file? I hope based on the environment variables the OpenMRS base image would dynamically generate a properties file.

Reply:
@mohan-13, yes this file was getting created dynamically, but was not having ```zeroDateTimeBehavior\=convertToNull``` this parameter in connection url, and it was causing an exception, that's why we added this file

Reply:
Please create a card for this and mark this as high priority @rohit-yawalkar .. we should take this up and bring inline with the rest of configuration settings asap - for the time being (dev only), its ok

Reply:
May be we can template this file using environment variable references and use the `envsubst` command to load values into the template at runtime. The same has been done for atomfeed.properties file in startup.sh